### ISSUE TITLE
Change your icon

### ISSUE THREAD

**Issue by nirav** [chenasraf/gInbox#28]
You should change your icon to this one since it's more appropriately sized.

https://dl.dropboxusercontent.com/u/18443/Inbox.icns

Your icon is way bigger on the dock in Yosemite.

Thanks for making this! Do you accept pull requests?

**Comment by chenasraf** [comment]
I do, feel free to create one :)

**Comment by jameswilson** [comment]
I agree with this post, but dont like the white background around the icon.   The letter needs to be sized down a bit on the larger version of the icon to add some whitespace around it.   This can be seen clearly if you use `CMD` + `TAB` to tab through the icons when the bezel highlights the icon, most icons have about 20 pixels of whitespace around them.

**Comment by jameswilson** [comment]
I've created a pull request based on the following icon:

ICNS: https://www.dropbox.com/s/f07jff4c2wv0rgl/AppIcon.icns?dl=0
Sketch: https://www.dropbox.com/s/6bqeiehdhz18aso/AppIcon.sketch?dl=0
